frontend/src/views/DifferentialExpression/components/Main/components/DeResults/components/DifferentialExpressionResults/connect.ts

```diff
@@ -98,5 +110,11 @@ export const useConnect = ({
     pageCount,
     handlePageChange,
     overlapPercent,
+    numDatasets1: `${datasets1.length} dataset${
+      datasets1.length !== 1 ? "s" : ""
```

Comment:
do we need to document what `"s"` means? At first glance given the var name `numDatasets1` I'd expect a number?